specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManager.json

```diff
@@ -0,0 +1,710 @@
+          "$ref": "#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/NetworkManagerNameParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "NetworkManagers_Get",
+        "description": "Get the specified Network Manager.",
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the network manager.",
+            "schema": {
+              "$ref": "#/definitions/NetworkManager"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkManagersGet": {
+            "$ref": "./examples/NetworkManagerGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "NetworkManagers_CreateOrUpdate",
+        "description": "Creates a Network Manager.",
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/NetworkManager"
+            },
+            "description": "Parameters supplied to the specify which Managed Network is."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated - Returns information about the network manager.",
+            "schema": {
+              "$ref": "#/definitions/NetworkManager"
+            }
+          },
+          "201": {
+            "description": "Created - Returns information about the network manager.",
+            "schema": {
+              "$ref": "#/definitions/NetworkManager"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Put Network Manager": {
+            "$ref": "./examples/NetworkManagerPut.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetworkManagers"
+        ],
+        "operationId": "NetworkManagers_Delete",
+        "description": "Deletes a network manager.",
+        "responses": {
+          "200": {
+            "description": "Delete Succeed."
+          },
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed."
```

Comment:
Specify the error schema here for consistency.
            "schema": {
              "$ref": "./network.json#/definitions/CloudError"
            }